### DISCUSSION TITLE
Windows Defender detections on ExplorerPatcher updaters

### DISCUSSION THREAD

**Discussion by nrose1124** [valinet/ExplorerPatcher#3523] [Q&A] [answered]
Latest update and several immediately preceding ones have tripped Windows Defender.

The latest tripped it for Bladabindi!ml, which is more than I want to just let pass freely.

Neither Defender nor MBAM has detected anything untoward after installation, of course.

Do we know what's going on here?

![image](https://github.com/user-attachments/assets/866c76c5-898f-4644-836c-8c67db0f1dfd)

**Comment by Amrsatrio** [comment, marked as answer]
Microsoft does not like EP and trained its ML models to block new variants of ep_setup.exe. That's why I said in the changelogs to add EP's folders into exclusions to prevent issues with updates (of course if you trust EP's maintainers such as me).

That's all I can comment.

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.
